integrations/fetch-job-inputs/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@ctrlplane/fetch-job-inputs",
+  "description": "GitHub Action to fetch variables from an Ctrlplane API",
+  "version": "1.0.0",
+  "keywords": [
+    "GitHub",
+    "Actions",
+    "Ctrlplane"
+  ],
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "packageManager": "pnpm@9.7.1",
+  "engines": {
+    "node": "20"
+  },
+  "type": "module",
+  "main": "index.ts",
+  "scripts": {
+    "dev": "pnpm exec tsx src/index.ts",
+    "build": "ncc build src/index.ts -o dist",
+    "build:action": "ncc build src/index.ts -o ../../github/fetch-job-inputs",
+    "lint": "eslint",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "@actions/core": "^1.10.1",
+    "@ctrlplane/node-sdk": "workspace:*",
+    "p-retry": "^6.2.0"
+  },
+  "devDependencies": {
+    "@ctrlplane/eslint-config": "workspace:*",
+    "@ctrlplane/prettier-config": "workspace:*",
+    "@ctrlplane/tsconfig": "workspace:*",
+    "@types/node": "catalog:node20",
+    "@vercel/ncc": "^0.38.1",
+    "eslint": "catalog:",
+    "tsx": "^4.19.1",
```

Comment:
where do we use this

Reply:
Lemme move them to dev deps. Good catch.